### PR TITLE
Some small bug fixes

### PR DESCRIPTION
**Pull request by flaxues**

### CODE REVIEW

File: src/BlazingRodGun/BlazingRodGun.php [hunk 1]
```diff
@@ -103,14 +103,14 @@ public function onItemHeld(PlayerItemHeldEvent $ev)
...
-			$ev->getPlayer()->sendTip(TextFormat::BLUE . "Vote GUN");
+			$ev->getPlayer()->sendTip(TextFormat::GOLD . "Vote GUN");
 		}
 		if ($ev->getPlayer()->getInventory()->getItemInHand()->getId() === 351) {
-			$ev->getPlayer()->sendTip(TextFormat::BLUE . "Bullets");
+			$ev->getPlayer()->sendTip(TextFormat::RED . "Bullets");
 		}
 	}
 
 	public function onFish(PlayerFishEvent $event){
```
Comment (thebigsmileXD) [inline src/BlazingRodGun/BlazingRodGun.php]: PlayerFishEvent? Is there any fishing done? Just curious ^^

**Comment by matthew4444** [comment]
hey ur on

**Comment by matthew4444** [comment]
its 2:13 am where u am lol

**Comment by thebigsmileXD** [comment]
8:16

**Comment by matthew4444** [comment]
ok

**Comment by matthew4444** [comment]
hey

**Comment by matthew4444** [comment]
https://gitter.im/matthew4444/NetherManiacsKingdom

**Comment by thebigsmileXD** [comment]
@matthew4444 Sorry, i don't use gitter

**Comment by matthew4444** [comment]
ok

**Comment by matthew4444** [comment]
im chattting with someone on gitter lol

**Comment by matthew4444** [comment]
aka

**Comment by matthew4444** [comment]
erkham2002

**Comment by matthew4444** [comment]
hey http://nethermaniacskingdom.freeforums.net/

**Comment by matthew4444** [comment]
hey]

**Comment by matthew4444** [comment]
i fixed the image from being large at the bottom here
http://nethermaniacskingdom.freeforums.net/

**Comment by matthew4444** [comment]
hey

File: src/BlazingRodGun/BlazingRodGun.php (hunk 1 above)
Comment (matthew4444) [inline src/BlazingRodGun/BlazingRodGun.php]: idk

Comment (matthew4444) [inline src/BlazingRodGun/BlazingRodGun.php]: that would be nice,but i think the code is 90% done lol ^^

**Comment by flaxues** [comment]
The event is in Genisys :P

**Comment by matthew4444** [comment]
hhey

**Comment by matthew4444** [comment]
im invieting a person

**Comment by matthew4444** [comment]
DavidJBrockway

File: src/BlazingRodGun/BlazingRodGun.php (hunk 1 above)
Comment (flaxues) [inline src/BlazingRodGun/BlazingRodGun.php]: The event is in Genisys :P

**Comment by DavidJBrockway** [comment]
Hey

**Comment by matthew4444** [comment]
hey

**Comment by matthew4444** [comment]
hey

**Comment by matthew4444** [comment]
@erkham2002 is coming on

**Comment by matthew4444** [comment]
hey
whats up

**Comment by DavidJBrockway** [comment]
Can you help fix skywars erro is on It on line 284 to 294 in https://github.com/DavidJBrockway/SkyWars-old-mg76-/blob/master/src/mcg76/skywars/map/SkyBlockGenerator.php

**Comment by matthew4444** [comment]
ill try

**Comment by DavidJBrockway** [comment]
I am so

**Comment by matthew4444** [comment]
ok

**Comment by flaxues** [comment]
Can you atleast tell the error ?

**Comment by DavidJBrockway** [comment]
my friend is working as well

**Comment by matthew4444** [comment]
k

**Comment by DavidJBrockway** [comment]
When fixed we have best skywars

**Comment by matthew4444** [comment]
here a crash report

**Comment by matthew4444** [comment]
Genisys Crash Dump Sun Feb 21 14:32:46 UTC 2016

Error: Declaration of mcg76\skywars\map\SkyBlockGenerator::init(pocketmine\level\generator\GenerationChunkManager $level, pocketmine\utils\Random $random) must be compatible with pocketmine\level\generator\Generator::init(pocketmine\level\ChunkManager $level, pocketmine\utils\Random $random) File: /Darkcraft_SkyWars_v1.0.4.phar/src/mcg76/skywars/map/SkyBlockGenerator Line: 294 Type: E_COMPILE_ERROR

THIS CRASH WAS CAUSED BY A PLUGIN 此次出错由插件引起

Code: [285] $this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () ); [286] foreach ( $this->populators as $populator ) { [287] $this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () ); [288] $populator->populate ( $this->level, $chunkX, $chunkZ, $this->random ); [289] } [290] } [291] public function getSpawn() { [292] return new Vector3 ( 132, 57, 129 ); [293] } [294] } [295] [296] [297] [298] [299] [300] [301] [302] [303] [304]

Backtrace:

**Comment by DavidJBrockway** [comment]
Ik it old fix that erro

**Comment by DavidJBrockway** [comment]
Now there a erro on the line I send in map skyblockgenrator

**Comment by matthew4444** [comment]
@flaxues u on?

**Comment by flaxues** [comment]
That's a easy fix but I'm at school right now

**Comment by flaxues** [comment]
So I can't

**Comment by DavidJBrockway** [comment]
I'm there also a world edit erro working on fix almost done need to fix wand fixed spleef

**Comment by matthew4444** [comment]
ok
will u fix it at https://github.com/NetherManiacsKingdom/SkyWars-old-mg76- when you get  back @flaxues ?

**Comment by DavidJBrockway** [comment]
I own a company called PocketServer I have xrnforo we should start our own pocketmine for us

**Comment by matthew4444** [comment]
how u have xenforo i need help with it?

**Comment by DavidJBrockway** [comment]
Then send to me plz I update mine then of that okay

**Comment by matthew4444** [comment]
ok iw ill

**Comment by matthew4444** [comment]
ok i will

**Comment by matthew4444** [comment]
promise

**Comment by DavidJBrockway** [comment]
__the game uses buttons to join a skywars map mcg76 said he stopes plugins

**Comment by matthew4444** [comment]
its 3 03 am
gtg at 5 30 am

**Comment by DavidJBrockway** [comment]
Cool I will fix world edit I try or you can

**Comment by matthew4444** [comment]
one sec

**Comment by matthew4444** [comment]
brb

**Comment by DavidJBrockway** [comment]
-Plugins I made TPW sign tp
pocketAntiFly ban ip none ope if they fly
PocketProtect stopes grifing

**Comment by DavidJBrockway** [comment]
I am a hacker in php I have full free versions Nulled i get the forum up ready to day we can start our own pocketmine forums with plugins

**Comment by DavidJBrockway** [comment]
I have Nulled loads of apps free xrnforo is working working on mine craft them

**Comment by matthew4444** [comment]
leakfourms.net

**Comment by matthew4444** [comment]
lol

**Comment by DavidJBrockway** [comment]
Nope

**Comment by matthew4444** [comment]
wht then

**Comment by DavidJBrockway** [comment]
Different I set it up after upgrade Nulled php ok we make our own pocketmine but need cool name I am a web Dev I made pocketserver.co.uk

**Comment by matthew4444** [comment]
\ok

**Comment by DavidJBrockway** [comment]
What name we should pick

**Comment by matthew4444** [comment]
is this for real
yayyayayayyayayyay free hosting

**Comment by DavidJBrockway** [comment]
Yes

**Comment by matthew4444** [comment]
finally after all these years

**Comment by DavidJBrockway** [comment]
I have a go daddy web server

**Comment by matthew4444** [comment]
hey

**Comment by DavidJBrockway** [comment]
Need cool name and new domain what souls we get

**Comment by matthew4444** [comment]
when i hit squid server it makes me go to top of page -.-

**Comment by matthew4444** [comment]
is tht a joke

**Comment by matthew4444** [comment]
-.-

**Comment by DavidJBrockway** [comment]
ehat

**Comment by DavidJBrockway** [comment]
What domain we gets

**Comment by matthew4444** [comment]
when i click squid it does this http://pocketserver.co.uk/mcpe-free-usd.html#

**Comment by matthew4444** [comment]
# what is this hastag for

**Comment by DavidJBrockway** [comment]
It still in beta

**Comment by matthew4444** [comment]
can u sign me up

**Comment by matthew4444** [comment]
with ftp acces plz

**Comment by DavidJBrockway** [comment]
Still add whmcs server peaches

**Comment by matthew4444** [comment]
if poosssbale

**Comment by matthew4444** [comment]
?

**Comment by DavidJBrockway** [comment]
We make out formusnlike pocketmjwn need cool name and I buy domain

**Comment by matthew4444** [comment]
hey can u just sign me up for this plan PocketServer Free MineCraft PE Hosting $0
SQUID EGG
Monthly Plan
10 Players
Ram 350 MB
DDoS Mitigation
Free Support
Unlimited Bandwidth

**Comment by matthew4444** [comment]
then just get 000webhost lol

**Comment by matthew4444** [comment]
i have a name

**Comment by erkam2002** [comment]
Hi

**Comment by DavidJBrockway** [comment]
Ok latter I re host server you need a domain and web server I got a web swerve that packages is MCPE free look at paid one I put free

**Comment by matthew4444** [comment]
EPIC MINECRAFT

**Comment by DavidJBrockway** [comment]
Can some one fix skywars now

**Comment by matthew4444** [comment]
EPICMINECRAFTHOST

**Comment by matthew4444** [comment]
.co.uk

**Comment by erkam2002** [comment]
HI

**Comment by matthew4444** [comment]
should be it

**Comment by matthew4444** [comment]
hey

**Comment by DavidJBrockway** [comment]
Spleef arena like old blsg spleef

**Comment by erkam2002** [comment]
free mcpe server?

**Comment by matthew4444** [comment]
ekrham2002 hey lets fix skywars https://github.com/DavidJBrockway/SkyWars-old-mg76-

**Comment by DavidJBrockway** [comment]
Ok i see end of this month then I get paid £70 every month

**Comment by matthew4444** [comment]
its in beta @erkam2002

**Comment by matthew4444** [comment]
ok

**Comment by matthew4444** [comment]
https://github.com/DavidJBrockway/SkyWars-old-mg76-

**Comment by matthew4444** [comment]
HEY LETS FIX THIS https://github.com/DavidJBrockway/SkyWars-old-mg76- EVERYONE!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by DavidJBrockway** [comment]
Ok what wrong with link

**Comment by erkam2002** [comment]
line nuber?

**Comment by erkam2002** [comment]
number*

**Comment by matthew4444** [comment]
@DavidJBrockway go to http://pocketserver.co.uk/mcpe-free-gbp.html and click on any plan, tell me what it does

**Comment by DavidJBrockway** [comment]
Need to go in school my links are working on my web server need to publish v8.1

**Comment by matthew4444** [comment]
ok

**Comment by matthew4444** [comment]
hey
need help

**Comment by matthew4444** [comment]
this messes up on my device

**Comment by matthew4444** [comment]
it doesnt do anything when i click on a plan

**Comment by matthew4444** [comment]
im on chrome latest

**Comment by matthew4444** [comment]
windows 10

**Comment by matthew4444** [comment]
http://pocketserver.co.uk/mcpe-free-gbp.html

**Comment by matthew4444** [comment]
@erkam2002 https://github.com/DavidJBrockway/SkyWars-old-mg76-/issues/1#issuecomment-187599212

**Comment by DavidJBrockway** [comment]
Line number https://github.com/DavidJBrockway/SkyWars-old-mg76-/blob/master/src/mcg76/skywars/map/SkyBlockGenerator.php Line It on line 284 to 294

**Comment by matthew4444** [comment]
public function populateChunk($chunkX, $chunkZ) {
		$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
		foreach ( $this->populators as $populator ) {
			$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
			$populator->populate ( $this->level, $chunkX, $chunkZ, $this->random );
		}
	}
	public function getSpawn() {
		return new Vector3 ( 132, 57, 129 );
	}
}

**Comment by matthew4444** [comment]
public function populateChunk($chunkX, $chunkZ) {
		$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
		foreach ( $this->populators as $populator ) {
			$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
			$populator->populate ( $this->level, $chunkX, $chunkZ, $this->random );
		}
	}
	public function getSpawn() {
		return new Vector3 ( 132, 57, 129 );
	}
}

**Comment by matthew4444** [comment]
public function populateChunk($chunkX, $chunkZ) {
		$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
		foreach ( $this->populators as $populator ) {
			$this->random->setSeed ( 0xdeadbeef ^ ($chunkX << 8) ^ $chunkZ ^ $this->level->getSeed () );
			$populator->populate ( $this->level, $chunkX, $chunkZ, $this->random );
		}
	}
	public function getSpawn() {
		return new Vector3 ( 132, 57, 129 );
	}
}

**Comment by matthew4444** [comment]
i see i think

**Comment by matthew4444** [comment]
$this->random->setSeed ( 0xdeadbeef
wht level is this in?
if u dont have tht level create it.

**Comment by thebigsmileXD** [comment]
Really.. this is so much spam..

**Comment by thebigsmileXD** [comment]
theres an edit comment button. even on phone

**Comment by matthew4444** [comment]
srlsy spam -.-
and there was always a edit comment button on phone -.- i have app

**Comment by thebigsmileXD** [comment]
@matthew4444 Get your password changed

**Comment by matthew4444** [comment]
WTF

**Comment by Skull3x** [comment]
Lmao tf

**Comment by flaxues** [comment]
Umm @matthew4444 wtf are you doing

**Comment by DavidJBrockway** [comment]
Did you fix skywars

**Comment by PocketKiller** [comment]
YEAH

**Comment by DavidJBrockway** [comment]
Cool I am Zena you nkwo

**Comment by DavidJBrockway** [comment]
Ok I upload my world edit

**Comment by matthew4444** [comment]
I changed my password,idk who made those dick msgs,i fixed it

**Comment by flaxues** [comment]
Well you edited everything...

**Comment by thebigsmileXD** [comment]
> I changed my password,idk who made those dick msgs,i fixed it

Good.. they changed my message from "password" to "dick"

**Comment by DavidJBrockway** [comment]
Can you fix world edit

**Comment by DavidJBrockway** [comment]
Ok GladitorMine like pocketmine /magicalmine we have a forums with plugin ext in 1 month and Jenkins working

**Comment by flaxues** [comment]
@DavidJBrockway WorldEdit need weakref which is not (yet) implemented in php7 version of pocketmine/genisys/clearsky etc.

**Comment by DavidJBrockway** [comment]
K

**Pull request merged**